packages/mds-transactions-service/@types/index.ts

```diff
@@ -1,20 +1,109 @@
 import { DomainModelCreate } from '@mds-core/mds-repository'
 import { RpcServiceDefinition, RpcRoute } from '@mds-core/mds-rpc-common'
+import { Nullable, Timestamp, UUID, VEHICLE_TYPE, VehicleEvent } from '@mds-core/mds-types'
+
+// one example -- many others are possible
+export interface TripReceiptDetailsDomainModel {
+  trip_id: UUID
+  start_timestamp: Timestamp
+  end_timestamp: Timestamp
+  vehicle_type: VEHICLE_TYPE
+  start_geography_id: Nullable<UUID>
+  end_geography_id: Nullable<UUID>
+  duration: Number // seconds
+  distance: Number // meters
+  trip_events: VehicleEvent[]
+}
+
+export interface CurbUseDetailsDomainModel {
+  trip_id: UUID
+  start_timestamp: Timestamp
+  end_timestamp: Timestamp
+  vehicle_type: VEHICLE_TYPE
+  geography_id: Nullable<UUID>
+  duration: Number // seconds
+  trip_events: VehicleEvent[]
+}
+
+export const FEE_TYPES = [
+  'base_fee',
+  'upgrade_fee',
+  'congestion_fee',
+  'trip_fee',
+  'parking_fee',
+  'reservation_fee',
+  'distance_fee',
+  'tolls_fee'
+] as const
+
+export type FEE_TYPE = typeof FEE_TYPES[number]
+
+export interface ReceiptDomainModel {
+  receipt_id: UUID
+  created: Timestamp // could be any time before the Transaction 'created'
```

Comment:
Any thoughts on just calling this (and other similar properties) `timestamp`?

Reply:
I thought about it.  No strong feeling either way.

Reply:
changed!